### PR TITLE
Make clearer why we think pyvusim is a reference simulator 

### PR DESCRIPTION
**Pull request by dannyjacobs**
Its not clear to the casual reader why the combination of analytic tests, continuous integration, reference simulations, documentation, community standards, and code publication adds up to pyuvsim being the root name certificate for interferometry simulators needing to demonstrate accuracy sufficient for 21cm cosmology. This pull request addresses that issue with edits to the README, addition of necessary memos, references to papers (some still in prep), and potentially, some re-organization of memos/reference simulations.

## Description
 - Update and expand the README, sharpening the language
 - TODO: add an overview memo describing analytic tests. (point sources, polarized point sources, diffuse sources)
 - TODO (maybe): Clean up the reference simulation directories. 

## Motivation and Context
 - Potential users are not aware of the plan and work behind pyuvsim
 - Addresses issue #312 
## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Reference simulation update or replacement
- [x] Documentation change (documentation changes only)
- [ ] Version change
- [ ] Build or continuous integration change

## Checklist:

Bug fix checklist:
- [ ] My fix includes a new test that breaks as a result of the bug (if possible).
- [ ] All new and existing tests pass.
- [ ] I have checked that I reproduce the reference simulations or if there are differences they are explained below (if appropriate). If there are changes that are correct, I will update the reference simulation files after this PR is merged.
- [ ] I have updated the [CHANGELOG](https://github.com/RadioAstronomySoftwareGroup/pyuvsim/blob/master/CHANGELOG.md).

New feature checklist:
- [ ] I have added or updated the docstrings associated with my feature using the [numpy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html).
- [ ] I have updated the documentation to highlight my new feature (if appropriate).
- [ ] I have added tests to cover my new feature.
- [ ] All new and existing tests pass.
- [ ] I have checked that I reproduce the reference simulations or if there are differences they are explained below (if appropriate). If there are changes that are correct, I will update the reference simulation files after this PR is merged.
- [ ] I have updated the [CHANGELOG](https://github.com/RadioAstronomySoftwareGroup/pyuvsim/blob/master/CHANGELOG.md).

Breaking change checklist:
- [ ] I have updated the docstrings associated with my change using the [numpy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html).
- [ ] I have updated the documentation to reflect my changes (if appropriate).
- [ ] My change includes backwards compatibility and deprecation warnings (if possible).
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests pass.
- [ ] I have checked that I reproduce the reference simulations or if there are differences they are explained below (if appropriate). If there are changes that are correct, I will update the reference simulation files after this PR is merged.
- [ ] I have updated the [CHANGELOG](https://github.com/RadioAstronomySoftwareGroup/pyuvsim/blob/master/CHANGELOG.md).

Reference simulation update or replacement:
- [ ] If this is a new reference simulation or if the settings files have changed, I have added all the relevant files to the reference_simulations folder.
- [ ] If this is a new reference simulation I have fully documented it in a memo in docs and updated any other docs as appropriate.
- [ ] I have updated the [CHANGELOG](https://github.com/RadioAstronomySoftwareGroup/pyuvsim/blob/master/CHANGELOG.md).

Documentation change checklist:
- [ ] Any updated docstrings use the [numpy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html).
- [ ] If this is a significant change to the readme or other docs, I have checked that they are rendered properly on ReadTheDocs. (you may need help to get this branch to build on RTD, just ask!)

Version change checklist:
- [ ] I have updated the [CHANGELOG](https://github.com/RadioAstronomySoftwareGroup/pyuvsim/blob/master/CHANGELOG.md) to put all the unreleased changes under the new version (leaving the unreleased section empty).
- [ ] I have run the appropriate tests for this level of version change (described in the readme under versioning).

Build or continuous integration change checklist:
- [ ] If required or optional dependencies have changed (including version numbers), I have updated the readme to reflect this.
- [ ] If this is a new CI setup, I have added the associated badge to the readme and to references/make_index.py (if appropriate).

### CODE REVIEW

**Comment by dannyjacobs** [comment]
Any reason not to merge in this doc update?

**Comment by dannyjacobs** [comment]
Errant text file deleted.
@bhazelton Did you say you had a comment on this PR? I don't see any, but its entirely possible I'm missing it.

**Review by bhazelton — COMMENTED** [review]
oops, apparently this was a pending comment rather than a posted one. Sorry!

File: README.md [hunk 1]
```diff
@@ -14,19 +14,35 @@ general simulator design.
 
 A comparison to other simulators may be found at [ReadTheDocs](https://pyuvsim.readthedocs.io/en/latest/comparison.html).
 
-## Motivation and Approach
-pyuvsim's two primary goals are interferometer simulation accuracy at the level of
-precision necessary for 21cm cosmology science, and maximum flexibility in use cases.
-Key elements of this approach include:
+## PYUVSIM, the Interferometer Simulator of Record
```
[the remark this answers is not in the document]
Comment (dannyjacobs) [reply README.md]: Sure, made lowercase.

**Review by bhazelton — APPROVED** [review]
Looks good.

**Pull request merged**